src/rpc/client.cpp

```diff
@@ -15,187 +15,118 @@
+    { "listunspent", {0, 1, 2} },
+    { "getblock", {1} },
+    { "getblockheader", {1} },
+    { "gettransaction", {1} },
+    { "getrawtransaction", {1} },
+    { "createrawtransaction", {0, 1, 2, 3} },
+    { "signrawtransaction", {1, 2} },
+    { "sendrawtransaction", {1} },
+    { "fundrawtransaction", {1} },
+    { "gettxout", {1, 2} },
+    { "gettxoutproof", {0} },
+    { "lockunspent", {0, 1} },
+    { "importprivkey", {2} },
+    { "importaddress", {2, 3} },
+    { "importpubkey", {2} },
+    { "verifychain", {0, 1} },
+    { "keypoolrefill", {0} },
+    { "getrawmempool", {0} },
+    { "estimatefee", {0} },
+    { "estimatepriority", {0} },
+    { "prioritisetransaction", {1, 2} },
+    { "setban", {2, 3} },
+    { "getspentinfo", {0} },
+    { "getaddresstxids", {0} },
+    { "getaddressbalance", {0} },
+    { "getaddressdeltas", {0} },
+    { "getaddressutxos", {0} },
+    { "getaddressmempool", {0} },
+    { "getblockhashes", {0, 1, 2} },
+    { "getblockdeltas", {0} },
+    { "zcbenchmark", {1, 2} },
+    { "getblocksubsidy", {0} },
+    { "z_listaddresses", {0} },
+    { "z_listreceivedbyaddress", {1} },
+    { "z_listunspent", {0, 1, 3} },
+    { "z_getaddressforaccount", {0, 1, 2} },
+    { "z_getbalance", {1, 2} },
+    { "z_getbalanceforaccount", {0, 1} },
+    { "z_getbalanceforaddress", {1} },
+    { "z_gettotalbalance", {0, 1, 2} },
+    { "z_mergetoaddress", {0, 2, 3, 4} },
+    { "z_sendmany", {1, 2, 3} },
+    { "z_shieldcoinbase", {2, 3} },
+    { "z_getoperationstatus", {0} },
+    { "z_getoperationresult", {0} },
+    { "z_importkey", {2} },
+    { "z_importviewingkey", {2} },
+    { "z_getpaymentdisclosure", {1, 2} },
+    { "z_setmigration", {0} },
+    { "z_getnotescount", {0} },
 };
 
-class CRPCConvertTable
-{
-private:
-    std::set<std::pair<std::string, int> > members;
-
-public:
-    CRPCConvertTable();
-
-    bool convert(const std::string& method, int idx) {
-        return (members.count(std::make_pair(method, idx)) > 0);
-    }
-};
-
-CRPCConvertTable::CRPCConvertTable()
-{
-    const unsigned int n_elem =
-        (sizeof(vRPCConvertParams) / sizeof(vRPCConvertParams[0]));
-
-    for (unsigned int i = 0; i < n_elem; i++) {
-        members.insert(std::make_pair(vRPCConvertParams[i].methodName,
-                                      vRPCConvertParams[i].paramIdx));
+set<int> paramsToConvertFor(const string& method) {
+    auto search = rpcCvtTable.find(method);
+    if (search != rpcCvtTable.end()) {
+        return search->second;
+    } else {
+        return set<int>();
     }
 }
 
-static CRPCConvertTable rpcCvtTable;
-
 /** Non-RFC4627 JSON parser, accepts internal values (such as numbers, true, false, null)
  * as well as objects and arrays.
  */
-UniValue ParseNonRFCJSONValue(const std::string& strVal)
+UniValue ParseNonRFCJSONValue(const string& strVal)
 {
     UniValue jVal;
-    if (!jVal.read(std::string("[")+strVal+std::string("]")) ||
+    if (!jVal.read(string("[")+strVal+string("]")) ||
```

Comment:
Ah, good. I prefer prefixes, too.

I also have a branch floating around that removes `using namespace` from all the headers that have it, because that is bad mojo.